schema/core.md

```diff
@@ -607,57 +609,66 @@ are used for a particular study.
 Note: `hasPart` and `isPartOf` are generally used for records that are grouped
 together or records that belong to other groupings. For example, `hasPart` may
 be used to indicate a record that consists of one or more additional
-CreativeWork (this is similar to a HydroShare collection). In contrast,
+`CreativeWork` (this is similar to a HydroShare collection). In contrast,
 `isPartOf` is used to indicate catalog records that reference a designated
-CreativeWork. As such, these are not demonstrated in the example below.
+`CreativeWork`. 
 
+The example below shows a simple schema for a `CreativeWork` that links to another
+`CreativeWork`.
 
 ``` json
 {
+  "@type": "CreativeWork",
+  "name": "Collection of Sample Datasets for IGUIDE",
+  "description": "A collection of sample datasets used by the I-GUIDE team",
+  "identifier": "https://my-unique-url.com/data-collection",
```

Comment:
I would avoid using ambiguous examples. This URL included in the identifier element does not look like an identifier to me.

Reply:
I agree. The original use of `url` was showing that the content may exist somewhere on the web. This is still acceptable to use in these examples, however `identifier` should be a unique ID or GUID that points to another entry in the catalog. The `identifier` property should be programmatically generated when the record saved into the catalog. We should explain this.